decentraland/unity-renderer
# fix: make unity editor work again with race condition fixes for .zone [MERGE ASAP TO UNBLOCK TEAM]

## What does this PR change?
@BrianAmadori @kuruk-mm there is something strange happening in that file. I left you a comment to apply your knowledge in the future and debug what is happening.

Here is a sequence diagram about how things should work between renderer and kernel.

![Screen Shot 2021-08-20 at 20 18 19](https://user-images.githubusercontent.com/260114/130301952-c0e8839c-1265-4d42-afb1-307557046700.png)
 

## How to test the changes?

Go to the editor and connect it to https://play.decentraland.zone, using the WSSController it should work now.

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
